doc/book/src/user/release-support.md

```diff
@@ -45,4 +45,5 @@ time, and may shift due to changes in network solution power.
 | 5.9.1-rc1 | 2024-05-22 | 2643624 | 2024-09-11 |
 | 5.9.1 | 2024-05-23 | 2644900 | 2024-09-12 |
 | 5.10.0-rc1 | 2024-08-22 | 2700600 | 2024-10-31 |
+| 5.10.0 | 2024-08-27 | 2706540 | 2024-11-05 |
```

Comment:
I checked that this is before the halving at 2726400 (it is ~17 days 6 hours before).